modulefiles/gfsutils_common.lua

```diff
@@ -5,7 +5,7 @@ Load common modules to build GFS utilities on all machines
 local netcdf_c_ver=os.getenv("netcdf_c_ver") or "4.9.2"
 local netcdf_fortran_ver=os.getenv("netcdf_fortran_ver") or "4.6.1"
 
-local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
+local bufr_ver=os.getenv("bufr_ver") or "12.0.1"
```

Comment:
This is the only thing that makes me pause.
Do we have `bufr_ver=12.0.1` installed as part of spack-stack 1.6.0 on other RDHPCS machines that EPIC maintains?  If not, then this change will break building of gfs-utils on Hera, Orion and Hercules.
I will check on WCOSS2, but the move to bufr v12 from v11 needs to be coordinated with the DA software as well.